index-es6.js

```diff
@@ -0,0 +1,693 @@
+/**
```

Comment:
So, do you think we should put this file in .gitignore? Do we really want to always remember to commit it?

Reply:
I think, it would be helpful to commit the generated file.